src/lib/throttle.js

```diff
@@ -132,3 +136,55 @@ export function getReturnToken() : ?string {
         return token;
     }
 }
+
+type CustomThrottle = Throttle & {
+    isActive : () => boolean
+};
+
+export function buildFundingLogoThrottle(props : Object) : CustomThrottle {
+
+    let emptyThrottle = {
+        isActive:     () => false,
+        isEnabled:    () => false,
+        isDisabled:   () => true,
+        getTreatment: () => '',
+        log:          () => emptyThrottle,
+        logStart:     () => emptyThrottle,
+        logComplete:  () => emptyThrottle
+    };
+
+    let { layout, label, locale, browserLocale } = props;
+
+    if (browserLocale.country !== COUNTRY.US) {
+        return emptyThrottle;
+    }
+
+    let localeString = locale;
+
+    if (typeof locale === 'object') {
```

Comment:
Locale prop will always be a string. Should be no need for this check.

Reply:
When you set `locale: 'en_US'` it actually shows up here as an object.  If `locale` is not set it defaults to the string `'en_US'`.

Reply:
Ah hmm, weird, that sounds like a quirk we need to fix on our side. Thanks for the heads-up. For now let's leave this check in that case.